lib/generators/sorcery/templates/initializer.rb

```diff
@@ -91,8 +91,12 @@
   # config.linkedin.key = ""
   # config.linkedin.secret = ""
   # config.linkedin.callback_url = "http://0.0.0.0:3000/oauth/callback?provider=linkedin"
-  # config.linkedin.user_info_mapping = {first_name: "firstName", last_name: "lastName"}
-  # config.linkedin.scope = "r_basicprofile"
+  # config.linkedin.user_info_mapping = {
+  #   first_name: 'localizedFirstName',
```

Comment:
Looking at the [docs](https://developer.linkedin.com/docs/fields/basic-profile), it looks like the fields were updated to `first-name` and `last-name`. I do remember these being nested hashes, so I wouldn't be surprised if `localizedXName` is what you're supposed to use. @JakubSzajna can you link to the documentation or otherwise confirm that this has been tested?

Reply:
Hey @athix, I was not able to find it anywhere in the docs, unfortunately, so I'm a little bit worried it might stop working at some point. But the code here is taken from my current working configuration, so yes, I've tested this and it works fine.

Reply:
Sounds good, it's broken as-is so we can go ahead and update it until LinkedIn decides to break it again. :shipit: